test/math/ExpDecayMath.ts

```diff
@@ -33,7 +33,7 @@ describe('ExpDecayMath', () => {
                 });
             } else {
                 await expect(expDecayMath.calcExpDecay(ethAmount, timeElapsed, halfLife)).to.revertedWithError(
-                    'Overflow'
+                    'panic code 0x11'
```

Comment:
This can be refactored to `revertedWithPanic` from [@nomicfoundation/hardhat-chai-matchers](https://www.npmjs.com/package/@nomicfoundation/hardhat-chai-matchers) (available through the toolbox or explicitly)

Reply:
should be addressed by Barak in his PR related to the exp decay math - and merged later into this branch

Reply:
@barakman FYI

Reply:
See https://github.com/bancorprotocol/carbon-contracts/pull/139#discussion_r1554954444 (decided to leave it as is and resolved the comment).

Reply:
@ivanzhelyazkov so you can go ahead and refactor this yourself?

Reply:
I do think here this should be related to a bigger refactor - waffle and hardhat-chai-matchers don't work with each other, and some packages need to be removed - as well as some changes made to the tests themselves - given that there's only a few stress tests which aren't run often, imo this change can be planned as a later refactor to the repo.